dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.41/V2_41_10__Add_missing_uid_indexes_interpretationcomment.sql

```diff
@@ -0,0 +1,20 @@
+/* add not null */
+UPDATE interpretationcomment SET uid = interpretationcommentid WHERE uid IS NULL;
```

Comment:
is interpretationcommentid a valid UID? Alternatively there is a stored procedure for generating uid added by DHIS2 in any database already.